scss/atoms/typography/_typography.scss

```diff
@@ -12,11 +12,11 @@
 body {
 	font: $normal #{ map-get( $defaultFont, font-size ) }/1.5 $sans;
 
-	@media ( max-width: 800px ) {
+	@media ( min-width: $viewport-xs ) {
```

Comment:
@arschmitz well now I understand why you were insisting on viewport, looking at it from that angle we should have "viewport-xs-min & viewport-xs-max" and so on, a total of 8 variables. I just thought we could achieve the same with only 4 breakpoint variables used to separate viewports insteak of explixitly stating their boundries. Since most people seem to agree on the 'viewport' approach we should adopt that